src/inputType/PanInput.ts

```diff
@@ -236,20 +239,64 @@ export class PanInput implements IInputType {
 	protected onHammerInput(event) {
 		if (this.isEnable()) {
 			if (event.isFirst) {
-				this.observer.hold(this, event);
+				this.panFlag = false;
+
+				if (event.srcEvent.cancelable !== false) {
+					this.observer.hold(this, event);
+					this.isRightEdge = IS_IOS_SAFARI && event.center.x > window.innerWidth - EDGE_DISTANCE;
+					this.panFlag = true;
```

Comment:
The name `panFlag` looks not meaningful. How about naming it to its purpose? Like, `scrolling` or so?

Reply:
@WoodNeck 

A `panmove` occurs even after a forced release from the axes, but a `change` event occurs by accepting it in observer.change.